processing/data_collection/gazette/pipelines.py

```diff
@@ -26,6 +31,9 @@ def process_item(self, item, spider):
         try:
             session.add(gazette)
             session.commit()
+        except IntegrityError as exc:
+            logger.warning(exc)
```

Comment:
You can use the logger from the spider. Something like this `spider.logger.warning `
spider.https://docs.scrapy.org/en/latest/topics/spiders.html#scrapy.spiders.Spider.logger

Reply:
Thanks, @jvanz.

I changed that in 8e399fe and also created two levels of logging to be more concise.

Reply:
I'm thinking about the debug logging, and I don't think we needed it because the warning about the duplication should be enough to understand what happens. What do you think?

Reply:
The trace is always useful for debugging. I liked what you did. If the developers want to see the trace they can enable the debug.